examples/tls_client_context_openssl.cc

```diff
@@ -82,7 +82,8 @@ int TLSClientContext::init(const char *private_key_file,
     return -1;
   }
 
-  SSL_CTX_set_default_verify_paths(ssl_ctx_);
+  /*SSL_CTX_set_default_verify_paths(ssl_ctx_);*/
+  SSL_CTX_set_verify(ssl_ctx_, SSL_VERIFY_NONE, 0);
```

Comment:
Is this accidentally left?

Reply:
Sort of. 🙈

Let me verify what the current state here is. wolfSSL has cross-signing cert verification as an optional feature and maybe I had that not enabled when I tested this.